backend/services/notification_service.py

```
@@ -613,6 +633,172 @@ async def _delete_message(self, msg_id: int) -> None:
+        
+        try:
+            notification_type = metadata.get("notification_type", "unknown") if metadata else "unknown"
+            
+            # Get user info from tokens
+            token_user_info = self._get_user_info_from_tokens(recipients)
+            
+            # Capture event for each recipient
+            for token in recipients:
+                user_info = token_user_info.get(token, {})
+                user_id = user_info.get("user_id")
+                email = user_info.get("email")
```

Ruff (0.14.14)

[error] Local variable `email` is assigned to but never used

Remove assignment to unused variable `email`

(F841)